### DISCUSSION TITLE
Windows 11 24H2

### DISCUSSION THREAD

**Discussion by kyrcy** [valinet/ExplorerPatcher#3885] [General]
I still see "We DO NOT recommend using EP on work machines running Windows 11 version 24H2." for the current latest release (22621.4317.67.1), but some of the listed OS builds tested are 24H2 and under more info, Windows 11 24H2/Canary (25314+) is also listed as supported. Can anyone clarify this? Is Windows 11 24H2 (10.0.26100.2033) supported?

**Comment by pyrates999** [comment]
Yes, it means don't run EP on work PC's.  Only on home PC's because that is not an environment where you care about uptime.

**Reply by kyrcy** [reply]
Do you recommend  using EP on work machines running Window 11 version 21H2, 22H2 and 23H2 then?

**Reply by Amrsatrio** [reply]
Yes, if you use the built-in Windows 10 taskbar. My point is I do not want to be liable for denial of service or productivity reduction due to EP's custom 10 taskbar implementation interfering with work. I don't want to cause anyone to get fired due to this as well.

Sure, the custom taskbar is stable today in 26100.2033, but some day in the future it might not be until I release an update if that happens.

**Reply by Amrsatrio** [reply]
And if you are on 22H2 or 23H2, I recommend to freeze your system on your current major version. If you have permissions, use the "Target feature version" group policy trick (Google it). By using 24H2 and continuing to use EP you will be subject to more frequent breakages arriving in a monthly basis.

**Reply by kyrcy** [reply]
I am not on on 22H2 or 23H2, so I guess I will not risk using EP even on home machines. Last time I tried it, taskbar stopped working properly and I had to reinstall Windows to solve the problem.

**Reply by Amrsatrio** [reply]
In what way it stopped working properly? Explorer didn't start up? The taskbar showed but didn't respond? Did you try following [this](https://github.com/valinet/ExplorerPatcher/wiki/ExplorerPatcher's-taskbar-implementation#help-my-screen-is-black)? Did you add the exclusions properly and as instructed?

I take major taskbar problems on 24H2 seriously so would be good if you can help on this one.

**Reply by kyrcy** [reply]
Explorer worked, but I had issues with the taskbar, for example clicking on the notification icon and on the time/date did nothing, so I could not open the notifications or the calendar. Nothing helped, this is why I had to reinstall Windows.

**Reply by Amrsatrio** [reply]
In that case there's an issue with your installation and it's unrelated with the taskbar. Good decision. Those flyouts are managed by ShellExperienceHost.exe which by itself is a UWP app, and has been known to be problematic at times.

**Reply by kyrcy** [reply]
Are you implying that the issues I had were not related with EP? Because I had no issues before installing it.

**Reply by Amrsatrio** [reply]
Personally I don't experience issues, even on the latest 24H2 Dev build. I wouldn't release updates that break the flyout at least on my real machines and VMs.

But what I can tell you is that if you enable "XAML UI sounds", the chance of those flyouts appearing successfuly would reduce to like 80%. If you did not enable it then I don't know what caused them to fail.

**Reply by Amrsatrio** [reply]
Back to your first question in this comment's replies.

> Tested on OS builds 22621.3296, 22631.4391, 22635.4440, 26120.961, 26100.1150, 26100.2161, and 26120.2213.

Yes, I've tested 67.1 on those builds, major features work fine as intended and advertised. Your build was on 26100.2033, you should've updated.

**Reply by kyrcy** [reply]
I have no idea what "XAML UI sounds" are and I am asking about LTSC if it matters.

**Reply by Amrsatrio** [reply]
Being LTSC or not should not matter. If it works in latest Dev and latest Release, then it should work on LTSC.

**Comment by Mixtzinn** [comment]
Mine is both a home and work PC because I am a translator and always work remotely. I have been using ExplorerPatcher without any issues for a long time. Today, my Windows 11 PC warns me that it cannot update to 24H2 because there is a program which is interfering, and recommends to update such program. Of course the program in question is EP and it is updated, but the Windows Update still can't go ahead. So, I assume the only way I can do the Windows Update is uninstalling EP, and after Windows is updated, I won't be able to use EP, right?

**Reply by Amrsatrio** [reply]
You can still use EP. But you have to be prepared for broken stuff.

**Reply by Mixtzinn** [reply]
Oh, that's a shame. Any clues about what kinds of broken stuff?

**Reply by Amrsatrio** [reply]
- 10 battery flyout, can be worked around by changing to 7 battery flyout. Not the same but it works.
- App names in apps that use legacy tray icons to send notifications do not show up.
- Simple Window Switcher doesn't work (due for fixing).
- Classic Alt Tab is gone, nuked from the OS. Important if explorer.exe isn't running. Not directly related to EP but a major pain point for me.
- In the future: Since the 10 taskbar and the 10 start menu are no longer part of the system, monthly updates have a very high tendency of breaking them. For EP's own taskbar that can cause black screen due to explorer.exe unable to start up. I usually respond fast to fix them, but not everyone knows how troubleshoot without doing a clean install. EP's taskbar works on latest 27xxx Canary build today, but may not be the case in a month.

**Reply by Mixtzinn** [reply]
Thank you very much for your quick clarifications. I will avoid mixing EP with 242H. One last question: do you see any chance of having a working EP version for 242H in the future?

**Reply by Amrsatrio** [reply]
I didn't say that EP does not work on 24H2, it works but with several features not really important for me broken. If you're using a desktop PC, you should be fine. I daily drive a Snapdragon X Elite laptop running 24H2, so if breakages come there, I'll be quick to fix them. Don't worry, you can use it.

**Reply by Amrsatrio** [reply]
To answer your question, if you assume that a working EP is like back in 21H2 days when the taskbar was supposed to be officially supported but they changed ther mind, unfortunately that is no longer possible and will never be a thing anymore. The 10 taskbar has been removed in 24H2. The only way to use it is to use recreations such as Windows 10 (ExplorerPatcher) taskbar style.

**Reply by Mixtzinn** [reply]
My PC  is not desktop. It's a laptop connected to a big screen and also external keyboard and mouse. I use it as a sort of UCP.

**Reply by Amrsatrio** [reply]
Yeah then, change the battery flyout to Windows 7 and you should be good to go.

**Reply by Mixtzinn** [reply]
> To answer your question, if you assume that a working EP is like back in 21H2 days when the taskbar was supposed to be officially supported but they changed ther mind, unfortunately that is no longer possible and will never be a thing anymore. The 10 taskbar has been removed in 24H2. The only way to use it is to use recreations such as Windows 10 (ExplorerPatcher) taskbar style.

OK, a real shame. Thank you for all the good work.

**Reply by Mixtzinn** [reply]
> Yeah then, change the battery flyout to Windows 7 and you should be good to go.

I don't know how to do that.

**Reply by Amrsatrio** [reply]
> I don't know how to do that.

![image](https://github.com/user-attachments/assets/06f8d982-409a-4659-882f-a6b2fa3df2f9)

> OK, a real shame. Thank you for all the good work.

Yeah, no problem. I did what was needed to keep EP working as advertised. It's up to the users if they're willing to continue the fight against MS or giving in to their horrible UI decisions.

**Reply by Mixtzinn** [reply]
Thank you again. I will uninstall EP and then update W11 and re-install EP and see what happens.
The problem is that I tried not updating my previous PC with W10 and that didn't work out so well. So I have to keep updating Windows.

**Reply by Amrsatrio** [reply]
https://www.kapilarya.com/select-the-target-feature-update-version-in-windows-11-10

**Comment by Mixtzinn** [comment]
Btw if you could point me to the latest version of the EP exe file, I would be grateful. I see a lot of files out there and I'm confused.

**Reply by Amrsatrio** [reply]
https://github.com/valinet/ExplorerPatcher/releases

Pick the 67.1 one. Download ep_setup.exe. 

ep_setup_arm64.exe is for Snapdragon laptops.

**Reply by PeaceByJesus** [reply]
Yes, thank Mr. Valinet indeed, and I thank God for him,  as well as other developers who labor to restore and enhance lost abilities and functionality due to Windows developers removing or changing  functions that were not broken or deficient, and not actually provided. 

With the beloved EP, we can run the W/10 TB, plus OpenShell, 7+Taskbar Tweaker, and enable a multi-level TB, etc. Some images: https://peacebyjesuscom.blogspot.com/2024/11/windows-users-why-not-switch-to-linux.html

I am presently running W/11 23H2 22631:4391 

The question now is, where is a donate button? The laborer is worthy of his wages

**Reply by pyrates999** [reply]
7+ Taskbar Tweaker does not work with the windows 10 EP implemented taskbar.  Use windhawk mods instead.

Donate button is here: https://github.com/valinet/ExplorerPatcher

It's under releases:

![image](https://github.com/user-attachments/assets/fd5c76d2-4980-4e60-8f05-253e9ad9289c)

**Reply by PeaceByJesus** [reply]
7+ Taskbar Tweaker does  work for me  with W/10 TB, though I had to do [something   ](https://ramensoftware.com/7-taskbar-tweaker/comment-page-54#comment-14811 )to enable Firefox to stack,  besides setting “taskbar.grouping.useprofile” preference in firefox to true   in about:config. 
https://photos.app.goo.gl/pbXt2r6iMqmB141D7 
https://photos.app.goo.gl/QtfTL9wsvBBibxhe7
And thanks for the donate link! (Did it via email since I did not recognize EU figures)

**Reply by freddellmeister** [reply]
https://peacebyjesuscom.blogspot.com/2024/11/windows-users-why-not-switch-to-linux.html

Thank God you posted this blog to spread his love.. I could not comment on your site directly, but I would probably upgrade your machine to W11 23H2 soon.

**Reply by PeaceByJesus** [reply]
I am blessed to know you are blessed, to God be the glory.  But the PP [link   ](https://www.paypal.com/donate?business=valentingabrielradu%40gmail.com&no_recurring=0&item_name=ExplorerPatcher)  only offers to donate in Euros with its Euro figures, and while I could have done that via a [currency converter,](https://www.xe.com/currencyconverter/convert/?Amount=10&From=USD&To=EUR), instead I just sent it to your email. Hope that is OK, 

But my PC is W/11 23H2 - the 22H2 was a typo. 

I am wary of 24H2 due to MS dislike of tweakers (I like the the over[ 200 tweaks ](https://www.thewindowsclub.com/list-of-tweaks-available-in-uwt4)available in Ultimate Windows Tweaker 5), which only improve Windows, and I thank God for such and for those as yourself who develop them. Thanks again, and I have often recommend EP.
